### DISCUSSION TITLE
head vs. title and relative change in TOC

### DISCUSSION THREAD

**Discussion by cranmer** [executablebooks/meta#108] [Ideas]
My first stumbling block was related to title vs. header in my notebooks. I often started with ## instead of # because I liked the font size better, but that turned into a mess, so I had to edit several notebooks, but that's not a big deal.

I wonder if it would be possible to have the `_toc.yml` annotate a relative up/down of the chapter or section.

**Comment by mmcky** [comment]
hi @cranmer -- this is an interesting idea. I've added a link to this discussion to items being looked at in a review of `_toc.yml`.
